react/package.json

```diff
@@ -27,6 +27,14 @@
       "react-app/jest"
     ]
   },
+  "overrides": {
```

Comment:
At some point, you'll have to share with me how you figured out some of these package.json changes (unless it was "I copied and pasted from SO")

Reply:
1) This specific one I copy and pasted from stack overflow, but also those specific overrides show up as an issue with react-scripts as part of an insane `npm audit` result (npm audit attempts to *downgrade* react-scripts from version 5 to version 3).
2) adding the babel thing to devDependencies is something the output of webpack(?) explicitly tells you to do
3) Pattern had an interesting one where literally it was oscillating between two different sets of deps every time I ran `npm install` so eventually I just had to tell it to lock one set so it didn't keep doing that forever.